keybert/_model.py

```diff
@@ -38,7 +39,12 @@ class KeyBERT:
     </div>
     """
 
-    def __init__(self, model="all-MiniLM-L6-v2", llm: BaseLLM = None):
+    def __init__(
+        self,
+        model="all-MiniLM-L6-v2",
+        llm: BaseLLM = None,
+        backend: BaseEmbedder = None,
```

Comment:
There is no need to add a `backend` parameter since you can pass the `SentenceTransformerBackend` directly to `model`. So there are no changes needed to this class to support it. 

There is a utility called `select_backend` that chooses the right backend based on the `model` parameter: 

https://github.com/MaartenGr/KeyBERT/blob/17330320789088338a08692ef866e0b07d861de7/keybert/backend/_utils.py#L12-L14

This means that there is no need to create that if/else statement to select the right backend as it does so automatically.